configure.ac

```diff
@@ -89,8 +89,8 @@ if test "$want_font_engine" = yes; then
 		AC_CHECK_LIB([freetype], [FT_Init_FreeType], [
 			want_font_engine=yes
 			font_engine_name=freetype2
-			LCUI_LIBS="$LCUI_LIBS `pkg-config --libs freetype2`"
-			CFLAGS="$CFLAGS `pkg-config --cflags-only-I freetype2`"
+			LCUI_LIBS="$LCUI_LIBS `pkg-config --libs freetype2` `pkg-config --libs fontconfig`"
+			CFLAGS="$CFLAGS `pkg-config --cflags-only-I freetype2` `pkg-config --cflags-only-I fontconfig`"
```

Comment:
I think you should add a check for fontconfig, like this:

```
want_font_config = yes
AC_ARG_ENABLE(font-config, AC_HELP_STRING([--enable-font-config],
[turn on font-config [[default=yes]]]), [want_font_config=$enableval])
if test "$want_font_engine" = yes; then
        AC_CHECK_LIB([freetype], [FT_Init_FreeType], []
        ......
        AC_CHECK_LIB([fontconfig], [......], [......])
fi
```

The result of the `./configure` script can be like this:

``` text
Build with lcui-builder support .... : yes
Build with libpng support .......... : yes
Build with libjpeg support ......... : yes
Build with font-engine support ..... : freetype2
Build with font-config support ..... : fontconfig
Build with thread support .......... : pthread
Build with video support ........... : x11
```